src/model/cloud-runner/cloud-runner-options.ts

```diff
@@ -220,21 +220,19 @@ class CloudRunnerOptions {
   }
 
   static get watchCloudRunnerToEnd(): boolean {
-    const input = CloudRunnerOptions.getInput(`watchToEnd`);
-
-    return input !== 'false';
+    return (CloudRunnerOptions.getInput(`watchToEnd`) || true) !== 'false';
   }
 
   public static get useSharedLargePackages(): boolean {
-    return CloudRunnerOptions.getInput(`useSharedLargePackages`) || false;
+    return (CloudRunnerOptions.getInput(`useSharedLargePackages`) || false) !== 'false';
```

Comment:
Should this be quoting the false like so?
```suggestion
    return (CloudRunnerOptions.getInput(`useSharedLargePackages`) || 'false') !== 'false';
```